extra/pylint.conf

```diff
@@ -9,7 +9,7 @@ reports=no
 disable=C0102,C0103,C0111,C0112,C0121,C0202,C0203,C0302,C0321,C0322,C0323,C0324,C0330,E0012,E1103,I0011,I0012,R0201,R0401,R0801,R0901,R0902,R0903,R0904,R0911,R0912,R0913,R0914,R0915,R0921,R0922,R0923,W0142,W0232,W0603
 
 [TYPECHECK]
-ignored-classes=Buffer,Caps,Frame,Client,Event,gi.repository.Gst,MainLoop,numpy,numpy.linalg,Sample,scipy.interpolate,scipy.optimize,_socketobject
+ignored-classes=Buffer,Caps,cec,Frame,Client,Event,gi.repository.Gst,MainLoop,numpy,numpy.linalg,pytest,Sample,scipy.interpolate,scipy.optimize,_socketobject
```

Comment:
Seems the only conflict is here, maybe I can help a bit? Just add cec and pytest into it right? Can we have this PR merged?
[pylint.zip](https://github.com/stb-tester/stb-tester/files/568551/pylint.zip)